### DISCUSSION TITLE
Flash U2W (Factory) to Autokit (Android Based)

### DISCUSSION THREAD

**Discussion by jonasius** [ludwig-v/wireless-carplay-dongle-reverse-engineering#94] [Q&A] [answered]
Hello,  

I ordered a Autokit Unit to use with android, but I got a U2W Plus instead 😠.
I saw on Issue https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/issues/8 that you @jsm174 tried to switch your device type. I wonder if you were successful? Did you have any other insights?

**Comment by jsm174** [comment, marked as answer]
@jonasius, Unfortunately no. I worked on it for quite a bit with no luck. Extremely frustrating since I bricked one, and made my other one completely non-upgradable (unless I replace the chip, or figure out how short the xtal of the cpu).
